apache/trafficcontrol
# Use ATS 8.1 in CDN in a Box

## What does this PR (Pull Request) do?


- [x] This PR fixes #5918 by using a Traffic Server RPM built for the latest version of CentOS 8 and getting the *CDN-in-a-Box CI* GitHub Actions workflow to pass.
- [x] This PR fixes #5919 by making CDN in a Box use a `trafficserver` RPM built from the 8.1.x ATS release branch.

## Which Traffic Control components are affected by this PR?


- CDN in a Box
- Documentation
- GitHub Actions
- T3C integration tests

## What is the best way to verify this PR?

1.  Build and run CDN in a Box, ensure caches are reported as healthy on http://trafficmonitor.infra.ciab.test/
    ```shell
    make -j6
    docker-compose build --parallel
    docker-compose kill && docker-compose down -v && docker-compose up -d
    docker-compose logs -f trafficmonitor
    ```

2.  Build and run CDN in a Box for CentOS 7, ensure caches are reported as healthy on http://trafficmonitor.infra.ciab.test/
    ```shell
    export RHEL_VERSION=7
    make -j6
    docker-compose build --parallel
    docker-compose kill && docker-compose down -v && docker-compose up -d
    docker-compose logs -f trafficmonitor
    ```

3.  Check the result of the *CDN-in-a-Box CI* GitHub Actions workflow

## If this is a bug fix, what versions of Traffic Control are affected?

 
 - master (5ad059f0af)
 - 5.1.2

## The following criteria are ALL met by this PR
